web/website/content/faq.md

```diff
@@ -6,11 +6,16 @@ Here are some of the most common questions we hear. Have something else you'd
 like to ask? Pop by our [Discord](https://discord.com/invite/eQcfaCmsNc) and ask
 away!
 
-{{< faq "Cool story Hansel, but what can I actually do with PRQL now?" >}}
+{{< faq "Cool story Hansel, but what can I actually do with PRQL?" >}}
 
-We're still early, and the opportunities for using PRQL are focused on two
-integrations:
+PRQL is a great tool for building SQL queries. In addition, we are focusing on
+several integrations. Here's what you can do right now:
 
+- **Use PRQL to query your database** Type your PRQL query into the
+  [Playground](https://prql-lang.org/playground/) and copy/paste the resulting
+  SQL into your database. Read our
+  [blog post](https://prql-lang.org/posts/2023-01-28-format-pretty-reports/) to
+  see how to make the reports more attractive.
```

Comment:
I don't think copying & pasting to write queries is a great product and, if my claim is correct, then us recommending it suggests we're not really in tune with users. The playground is great for exploring the language, but it's not good for querying data.

Very happy to find a different way of linking to the blog post though

Reply:
Yes, but... 

I think the Playground (and/or VS Code extension) are the most useful and easily accessible ways to use PRQL. _Anybody_ can copy/paste and do real work: that's how I use PRQL. That's why I listed them first. (I don't particularly care about the link to my blog post.)

As for the other bindings: they're powerful tools, but for very specific audiences. (I don't recognize either of these names, except that they show up in the PRQL FAQ :-)

I did a little rewording, but won't mind if someone makes further changes. I absolutely encourage others to add to the list of other tools that can use PRQL.

Reply:
We could add something like "...as well as prototyping queries in VS Code or the Playground". You can't really _query_ data there, which is important, but I agree it could still be mentioned.

I also think we could add a reference to the supported bindings such as `prqlr` here (CC @eitsupi )

We also could remove the dbt extension, but that only leaves us with Python! Lots to do here...

Reply:
Perhaps we could say that "we can use the bindings to create programs that uses PRQL instead of SQL"?

It would be nice if we could express that we can make PRQL work immediately with a thin wrapper like `pyprql.magic` or `prqlr`'s knitr engine.

Reply:
I think we should have somewhere where we list how folks can write PRQL without writing code to compile PRQL 

That's what most people want to do! it's a big gap for PRQL at the moment...

`prqlr` is a good library for that! I'd be very happy to mention it here

Reply:
I liked your thought about "prototyping the queries" - that's what's there now.